vnext/Shared/RuntimeOptions.cpp

```diff
@@ -42,4 +54,13 @@ const int32_t __cdecl GetRuntimeOptionInt(const string &name) noexcept {
   return 0;
 }
 
+const char *__cdecl GetRuntimeOptionString(const char *name) noexcept {
```

Comment:
Because of the intended ABI safety, we can not use `std::string`.
@vmoroz is there a known pattern to prevent accessing invalid memory when returning the char pointer?

Reply:
Chatted offline. Delivering a heap copy of the underlying char array.
Will add a doc comment indicating this `const char*` should be manually managed (deleted).

Reply:
Please see my other comment about this method per our the most recent discussion offline on 4/18/2022